al_aws_collector.js

```diff
@@ -378,6 +402,27 @@ class AlAwsCollector {
             }
         ],
         function(err, checkinParts) {
+
+            const invocationStatsDatapoints = checkinParts[1].statistics[0].Datapoints ? checkinParts[1].statistics[0].Datapoints : checkinParts[1].statistics;
+            const errorStatsDatapoints = checkinParts[1].statistics[1].Datapoints ? checkinParts[1].statistics[1].Datapoints : checkinParts[1].statistics ;
+            const collectorStreams = collector._streams;
+            if (checkinParts[0].status === 'ok'  && invocationStatsDatapoints.length > 0  && invocationStatsDatapoints[0].Sum > 0
+                 && errorStatsDatapoints.length > 0 && errorStatsDatapoints[0].Sum === 0) {
+                if (Array.isArray(collectorStreams) && collectorStreams.length > 0) {
+                    collectorStreams.map(streamType => {
+                        // make api call to send status ok
+                        let okStatus = collector.prepareHealthyStatus(`${collector._applicationId}_${streamType}`);
+                        collector.sendStatus(okStatus, () => {
```

Comment:
we should be able to send status updates for all streams in one `sendStatus()` request. Currently each stream status is sent separately.
Let's change `sendStatus()` a little bit to accept array of statuses to be transferred.

Reply:
send the array of status to sendStatus() to update all status in one api call.